Components/StockPage.js

```diff
@@ -100,7 +103,43 @@ export default function StockPage(props) {
     };
   }, []);
 
+  //trying to read favorites from DB on launch
+ 
+
+  const readFavoritesFromDB = () => {
+    let isMounted = true;
```

Comment:
I think it should be okay to readFavoritesFromDB into the isMounted statement in the first useEffect and get rid of the isMounted.